release/images.sh

```diff
@@ -36,6 +36,30 @@ sign_checksum() {
     fi
 }
 
+finalize_image() {
+    local name=${1}
+
+    local finalize_script=$(find "${TEMPLATES_PATH}" -name "${name}-finalize.*" -print -quit)
+    if [[ -n "${finalize_script}" ]]; then
+        local script_log="${LOG_DIR}/${name}-finalize"
+        "${finalize_script}" "${WORK_DIR}/${name}.img" > "${script_log}.log" 2> >(tee "${script_log}_err.log")
+
+        if [[ -s "${script_log}_err.log" ]]; then
+            log "Image '${name}'" "NOT OK!"
+            return 1
+        fi
+    fi
+
+    local images=$(ls "${WORK_DIR}/${name}".*)
+    local final_file="${IMGS_DIR}/${DISTRO_NAME}-${MIX_VERSION}-${name}"
+    for image in ${images}; do
+        local ext="${image#*.}"
+        mv "${image}" "${final_file}"."${ext}"
+        sha512sum "${final_file}.${ext}" > "${final_file}.${ext}-${CHKSUM_FILE_SUFFIX}"
+        sign_checksum "${final_file}.${ext}-${CHKSUM_FILE_SUFFIX}"
```

Comment:
`${final_file}.${ext}` is used a few times.  Maybe consider defining a local variable for image file path and reusing it?

Reply:
That has been renamed to use final_path & final_file variables